packages/eslint-config/react-internal.js

```diff
@@ -32,6 +32,7 @@ export const config = [
     settings: { react: { version: "detect" } },
     rules: {
       ...pluginReactHooks.configs.recommended.rules,
+      "react/prop-types": "off",
```

Comment:
What was this affecting? Would this be better handled by disabling the specific instances instead of turning the rule off everywhere?

Reply:
Ohhh, gotcha. I'm fine with turning them off then I think!